Throws a whole pile of Postgresql errors as database has not been initialized

The initial creation/migration does not get run leading to an unusable state.  Looking into where it needs to be done, but maybe you'd be faster?

```
web_1       | django.db.utils.ProgrammingError: relation "django_site" does not exist
web_1       | LINE 1: ..."django_site"."domain", "django_site"."name" FROM "django_si...
web_1       |                                                              ^
web_1       | 
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" WHERE UPPER("django_site"."domain"::text) = UPPER('192.168.99.100')
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" ORDER BY "django_site"."domain" ASC LIMIT 21
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" WHERE UPPER("django_site"."domain"::text) = UPPER('192.168.99.100') LIMIT 21
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" WHERE UPPER("django_site"."domain"::text) = UPPER('192.168.99.100') LIMIT 21
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" WHERE UPPER("django_site"."domain"::text) = UPPER('192.168.99.100') LIMIT 21
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" WHERE UPPER("django_site"."domain"::text) = UPPER('192.168.99.100') LIMIT 21
postgres_1  | ERROR:  relation "django_site" does not exist at character 78
postgres_1  | STATEMENT:  SELECT "django_site"."id", "django_site"."domain", "django_site"."name" FROM "django_site" WHERE UPPER("django_site"."domain"::text) = UPPER('192.168.99.100')
web_1       | [2016-08-14 14:30:43 +0000] [14] [ERROR] Error handling request
'''
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -2,6 +2,7 @@
 
 # A microservice for the web app.
 web:
+  container_name: web
   restart: always
   build: ./web
   expose:
@@ -16,6 +17,7 @@ web:
 
 # A microservice for the web server.
 nginx:
+  container_name: nginx
   restart: always
   build: ./nginx/
   ports:
@@ -30,6 +32,7 @@ nginx:
 
 # A microservice for the database.
 postgres:
+  container_name: postgres
   restart: always
   image: postgres:latest
   ports:

